--- installer.nsi	2020-06-30 15:02:06.993946000 +0200
+++ installer-n.nsi	2020-06-30 15:01:41.002224900 +0200
@@ -127,7 +127,7 @@
   ; The output path becomes the working directory for shortcuts
   SetOutPath "%HOMEDRIVE%\%HOMEPATH%"
     CreateShortCut "$SMPROGRAMS\Quango+.lnk" "$INSTDIR\Python\pythonw.exe" \
-      '"$INSTDIR\Quango+.launch.pyw"' "$INSTDIR\quango+.ico"
+      '"$INSTDIR\Quango+.launch.pyw" --tango-host=ictrlfs.ictrl.frm2:10000' "$INSTDIR\quango+.ico"
   SetOutPath "$INSTDIR"
 
 
