--- installer.nsi	2020-06-25 19:12:48.583930200 +0200
+++ installer-n.nsi	2020-06-26 08:18:34.345668900 +0200
@@ -127,7 +127,7 @@
   ; The output path becomes the working directory for shortcuts
   SetOutPath "%HOMEDRIVE%\%HOMEPATH%"
     CreateShortCut "$SMPROGRAMS\Charm-Mesytec-Histogram.lnk" "$INSTDIR\Python\pythonw.exe" \
-      '"$INSTDIR\Charm-Mesytec-Histogram.launch.pyw"' "$INSTDIR\quango-histogram.ico"
+      '"$INSTDIR\Charm-Mesytec-Histogram.launch.pyw" --tango-host=ictrlfs.ictrl.frm2:10000' "$INSTDIR\quango-histogram.ico"
   SetOutPath "$INSTDIR"
 
 
