--- installer.nsi	2020-09-29 15:25:03.832388100 +0200
+++ installer-b.nsi	2020-09-29 15:31:58.757183500 +0200
@@ -127,7 +127,7 @@
   ; The output path becomes the working directory for shortcuts
   SetOutPath "%HOMEDRIVE%\%HOMEPATH%"
     CreateShortCut "$SMPROGRAMS\QuangoPlus.lnk" "$INSTDIR\Python\pythonw.exe" \
-      '"$INSTDIR\QuangoPlus.launch.pyw"' "$INSTDIR\quango+.ico"
+      '"$INSTDIR\QuangoPlus.launch.pyw" --tango-host=ictrlfs.ictrl.frm2:10000' "$INSTDIR\quango+.ico"
   SetOutPath "$INSTDIR"
 
 
